### DISCUSSION TITLE
Are tikz images in the visual editor supposed to render a preview?

### DISCUSSION THREAD

**Discussion by jonas-karlsson** [quarto-dev/quarto-cli#6070] [Q&A] [answered]
### Description

Hi there,

I am inserting a tikz image (file ending `.tex`) in the visual editor using (Insert → Figure/Image) in RStudio.

When adding other images (png, svg etc) I get a preview of the image. With the tikz image, I do not get a preview, just this [broken image placeholder](https://pasteboard.co/JvbaCXUZUkr1.png).

The image shows up as expected when rendering to PDF.

Is the preview supposed to work? And if so, what could I be doing wrong?

Thanks for a great piece of software!

### Versions
RStudio Version 2023.06.0+421 (2023.06.0+421) 
quarto 1.3.433
MacOs Ventura 13.4.1 (22F82) on Intel.

**Comment by mcanouil** [comment, marked as answer]
I am not sure it's related to the visual editor or instead the IDE, but I would say it's an issue/question for <https://github.com/rstudio/rstudio/issues>.

**Reply by jonas-karlsson** [reply]
Okay. Will post there, thanks.
